functests/2_performance_update/updating_profile.go

```diff
@@ -55,6 +56,67 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	Context("Verify GloballyDisableIrqLoadBalancing Spec field", func() {
+		It("[test_id:36150] Verify that IRQ load balancing is enabled/disabled correctly", func() {
+			irqLoadBalancingDisabled := profile.Spec.GloballyDisableIrqLoadBalancing != nil && *profile.Spec.GloballyDisableIrqLoadBalancing
+
+			profileIsolatedCPUs := string(*profile.Spec.CPU.Isolated)
+			isolatedCPUsMask, err := components.CPUListToHexMask(profileIsolatedCPUs)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to list in Hex %s: %v", profileIsolatedCPUs, err))
+			isolatedCPUs, err := strconv.ParseUint(isolatedCPUsMask, 16, 64)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to parse %s: %v", isolatedCPUsMask, err))
+
+			var expectedBannedCPUs uint64
+			if irqLoadBalancingDisabled {
+				expectedBannedCPUs = isolatedCPUs
+			}
+
+			for _, node := range workerRTNodes {
+				bannedCPUs, err := nodes.BannedCPUs(node)
+				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to extract the banned CPUs from node %s: %v", node.Name, err))
+				Expect(bannedCPUs).To(Equal(expectedBannedCPUs))
+			}
+
+			By("Modifying profile")
+			initialProfile = profile.DeepCopy()
+
+			irqLoadBalancingDisabled = !irqLoadBalancingDisabled
+			if irqLoadBalancingDisabled {
+				expectedBannedCPUs = isolatedCPUs
+			} else {
+				expectedBannedCPUs = 0
+			}
+			profile.Spec.GloballyDisableIrqLoadBalancing = &irqLoadBalancingDisabled
+
+			By("Updating the performance profile")
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
```

Comment:
I think we need to wait here, don't we?

Reply:
Not really, we wait once for the node to have the right settings. Anything in between does not worth the wait.

Reply:
ok